AcsfToolsCommands.php

```diff
@@ -175,6 +175,8 @@ public function ml($cmd, $command_args = '', $command_options = '', $options = [
         unset($options['profiles']);
       }
 
+      $i = 0;
+      $delay = abs($options['delay']);
```

Comment:
I agree with Nikunj on the abs thing. Really needed?
Also make sure we don't have a notice if param is not sent. I guess no as there is a default value.

Reply:
`abs()` was here to handle the negative values as `sleep()` works only for positive. As now doing `> 0` test, no need it.

PR updated.